helm/cert-exporter/values.yaml

```diff
@@ -0,0 +1,6 @@
+namespace: kube-system
```

Comment:
I'm not sure this is the best way to fix this. As we have 2 charts with different values. Maybe with explanatory comments this is OK.

My thought was that we change the values structure and we also set it to `kube-system` in the default catalog. https://github.com/giantswarm/installations/blob/88cd79ef750338a48a554195bc148cbebaedbe40/default-default-appcatalog-values.yaml

```
exporter:
  namespace: kube-system
```
WDYT?